Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```diff
@@ -104,7 +104,7 @@
     visuals:
     - type: ComputerVisualizer
       key: security_key
-      screen: security
+      screen: explosive
```

Comment:
What's going on with this change

Reply:
The records computer's layer is called explosive, I don't know why.